Lib/_pycodecs.py

```
@@ -93,98 +128,101 @@
-                b.append(i%256)
+                b.append(i % 256)
                 i >>= 8
             if sys.byteorder == "big":
                 b.reverse()
             p += b
         res = bytes(p)
         return res, len(res)
     else:
-        res = "You can do better than this" # XXX make this right
+        res = "You can do better than this"  # XXX make this right
         return res, len(res)
```

Pylint (3.3.7)

[refactor] Unnecessary "else" after "return", remove the "else" and de-indent the code inside it

(R1705)